cmd/alert-agent/init/init.go

```diff
@@ -154,6 +155,35 @@ func Cmd() *cobra.Command {
 						Amount: amount,
 					},
 				})
+
+				if keyInfo.Name == consts.KeysIds.Eibc {
+					eibcHome := filepath.Join(home, consts.ConfigDirName.Eibc)
+					eibcConfigPath := filepath.Join(eibcHome, "config.yaml")
+					eibcConfig, err := eibcutils.ReadConfig(eibcConfigPath)
```

Comment:
check whether the eibc config exists, if it does not - skip the entire grants step